### ISSUE TITLE
Fix gisida-react staging build process

### ISSUE THREAD

**Issue by kahummer** [onaio/gisida#441]
When creating an optimized production build on staging we get this error:
![Screenshot from 2020-03-13 14-40-58](https://user-images.githubusercontent.com/16018123/76618098-c8d1fb80-6538-11ea-9ab6-0287bb1a7787.png)

**Comment by kahummer** [comment]
@ciremusyoka  @kelvin-muchiri Please update the issue once you get a fix to this I'm still researching on the same

**Comment by ciremusyoka** [comment]
This bug is caused by the way we reference `gatekeeper module`.

1.  `import { ... } from '@onaio/gatekeeper';`  produces the error.
2.  `import { ... } from '@onaio/gatekeeper/dist/components/login';` does not. 

Can't figure out why option `1` which is the right way is causing the error.

**Comment by ciremusyoka** [comment]
> @kahummer @kelvin-muchiri This bug is caused by the way we reference `gatekeeper module`.
> 
> 1. `import { ... } from '@onaio/gatekeeper';`  produces the error.
> 2. `import { ... } from '@onaio/gatekeeper/dist/components/login';` does not.
> 
> Can't figure out why option `1` which is the right way is causing the error.

@moshthepitt what is your thought on this?

**Comment by moshthepitt** [comment]
@ciremusyoka what is the specific thing that is being imported in this case?  It may be such that this thing is not exported from the index file at `'@onaio/gatekeeper'` - in which case we can update gatekeeper to export it.

**Comment by ciremusyoka** [comment]
@moshthepitt  `useOAuthLogin` and `AuthorizationGrantType`  which i believe they are at the index file of '@onaio/gatekeeper'

**Comment by kelvin-muchiri** [comment]
@ciremusyoka Yea, I remember discussing this with @p-netm he was supposed to create an issue for it. But I remember I used [@onaio/gatekeeper/dist/components/logi](https://github.com/onaio/gisida-react/blob/staging/src/lib/components/Login/OnaOauthLogin/OnaOauthLogin.js#L3)

Or is the issue being raised by [`import { AuthorizationGrantType } from '@onaio/gatekeeper';`](https://github.com/onaio/gisida-react/blob/staging/src/lib/components/Login/OnaOauthLogin/OnaOauthLogin.js#L4)?

**Comment by ciremusyoka** [comment]
@kelvin-muchiri  yes this `import { AuthorizationGrantType } from '@onaio/gatekeeper';` is what raised the issue.

**Comment by cKellyDesign** [comment]
Are we using the right version of gatekeeper?

**Comment by cKellyDesign** [comment]
@kahummer @ciremusyoka @kelvin-muchiri what's the status of this fix?
